core/src/main/java/org/infinispan/interceptors/base/CommandInterceptor.java

```diff
@@ -94,12 +92,6 @@ public final boolean hasNext() {
       return myIndex < interceptors.size();
    }
 
-   /**
-    * Does nothing since 9.0.
-    */
-   public final void setNext(CommandInterceptor ignored) {
```

Comment:
It's public API, I don't think we should delete it in 9.4. Same with `getLockAcquisitionTimeout()` and `getCacheWithFlags()`, there could be user subclasses using them in the wild.

Reply:
Ok, let's not upset 'wildlife' then :)